public/openapi.json

```diff
@@ -2911,6 +2911,18 @@
         "operationId": "devicesListDeviceProvidersPost"
       }
     },
+    "/internal/tlmtry": {
```

Comment:
lol we should probably exclude this, surprised internal endpoints are captured

Reply:
We exclude them / separate them in connect, but the fake should have all endpoints IMO here.

Reply:
woops didn't realize this was the fake :facepalm: